configure.py

```diff
@@ -1061,6 +1064,7 @@ def __init__(self, infofile):
                 'output_to_exe': '-o ',
                 'add_include_dir_option': '-I',
                 'add_lib_dir_option': '-L',
+                'add_sysroot_option': '--sysroot=',
```

Comment:
This shouldn't be a default; this causes it to be used for all compilers. For example AppVeyor (MSVC build) warns:

> cl : Command line warning D9002 : ignoring unknown option '--sysroot='

Instead, have it set to empty here, and then set the `add_sysroot_option` for GCC and Clang in the compiler info files (`src/build-data/cc/gcc.txt` and `clang.txt`)